pages/treeherder.py

```diff
@@ -61,6 +61,7 @@ class ResultSet(PageRegion):
 
         _datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span a')
         _jobs_locator = (By.CLASS_NAME, 'job-btn')
+        _pin_all_jobs_locator = (By.CSS_SELECTOR, '.result-set .glyphicon-pushpin')
```

Comment:
We should be able to just use `(By.CLASS_NAME, '.result-set .glyphicon-pushpin')` and find from the context of the root element.